packages/terra-abstract-modal/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Changelog
 Unreleased
 ----------
 
+Unreleased
```

Comment:
Whoops added Unreleased

Reply:
De-duped in this commit: https://github.com/cerner/terra-framework/pull/626/commits/b2ba9614b3f3761e7459d2f7efe6d5a9cf3d1d87